packages/flutter_tools/lib/src/commands/doctor.dart

```diff
@@ -14,6 +14,12 @@ class DoctorCommand extends FlutterCommand {
       negatable: false,
       help: 'Run the Android SDK manager tool to accept the SDK\'s licenses.',
     );
+    argParser.addFlag('verbose',
```

Comment:
There's already a global `'verbose'` flag in `FlutterCommandRunner` - does this mean these are now semantically different?

```
flutter doctor --verbose
flutter --verbose doctor
flutter --verbose doctor --verbose
```

Ok if that's intentional - I just want to make sure we're being explicit about it.

Reply:
IMHO we should just have the one `--verbose`/`-v`; see my comment above about `--verbose-logs`.

Reply:
Allowing flags to have aliasing like this is going to be a bit more complicated and I'd prefer to defer that to a follow up PR if that's acceptable

Reply:
Well we should definitely only have one "verbose" flag (the global one). I don't mind if we don't add `--verbose-logs` in this PR, and in the meantime just have "verbose" have special meaning for "doctor" the same way it does for "--help".

Reply:
Implemented in this PR as verbose having special meaning for 'doctor'.